src/test/java/com/zachary_moore/runner/LintRunnerShould.java

```diff
@@ -82,11 +87,195 @@ public void lintRunnerShouldTakeMultipleFiles() throws Exception {
         LintRunner lintRunner =
                 new LintRunner(
                         this.lintRegister,
-                        "./src/test/resources/test-2.json",
-                        "./src/test/resources/test-file.pdsc");
-        Map<LintRule, Map<JSONFile, List<String>>> lintOutput = lintRunner.lint();
+                    "./src/test/resources/test-2.json",
+                    "./src/test/resources/test-file.pdsc");
+        lintRunner.lint();
+
+        Map<LintRule, Map<JSONFile, List<LintError>>> lintOutput = lintRunner.getLintOutput();
 
-        assert(lintRunner.analyzeLintAndGiveExitCode() == 1);
+        assert(lintRunner.getExitCode() == 1);
         assert(lintOutput.get(lintRule).size() == 2);
     }
+
+    @Test
+    public void lintRunnerShouldValidErrorForWrappedPrimitive() throws Exception {
+        LintRule lintRule = new LintRule.Builder()
```

Comment:
Can pull out lintRule and hold on as a LintRule.Builder and only make implementation to avoid all the duplication in these tests